content/overview/getting-started/_index.en.md

```diff
@@ -5,50 +5,149 @@ weight: 2
+![REPL](../wash_up.png)
 
-# add js plugin
-[[params.plugins.js]]
-link = "define plugins path"
+To see a list of running hosts, run the following command in the REPL:
+```shell
+wash> ctl get hosts
+```
+You should see one host running, and you can view its inventory by running the following command with the **Host ID** found in the Output window:
+```shell
+wash> ctl get inventory NXYZABC...
+```
+You'll see output similar to the following (public key output truncated for documentation):
+```shell
+               Host Inventory (NBMRMKELSYA645ZBGN6AEYJU6GV...)                
+                                                                                                        
+hostcore.os                                                      macos                                        
+hostcore.osfamily                                                unix                                         
+hostcore.arch                                                    x86_64                                       
+repl_mode                                                        true                                         
+                          No actors found                                             
+                                                                                                        
+Provider ID               Link Name                  Image Reference   
+VDHPKGFKDI34Y4RN4PWWZH... default                    N/A               
+VAHNM37G4ARHZ3CYHB3L34... __wasmcloud_lattice_cache  N/A               
+```
+Currently on this Host, we have a few labels that show the environment this Host is running on, and two default providers. The first, starting with `VDHPK`, is the `extras` provider, which provides an interface for wasmCloud resources to generate random numbers, GUIDS, etc. The second, starting with `VAHNM`, is the lattice cache provider, which provides a distributed cache to all hosts running in this lattice. We'll talk more about what a **Lattice** is in the [Platform Builder](../../platform-builder) section of the documentation.
 
+#### Running an Actor
+We can start scheduling Actors and Providers right away on this Host using the `ctl start` command. To begin, we'll start our `Echo` sample actor.
+```shell
+wash> ctl start actor wasmcloud.azurecr.io/echo:0.1.0
+```
+The `Echo` sample actor has a single operation `HandleRequest`, which will respond to a delivered HTTP Request with an identical response that "echoes" the request sent.
 
-# main menu
-[[menu.main]]
-name = "contact"
-url = "contact"
-weight = 1
+#### Running a Capability Provider
+In order for this actor to receive HTTP requests, we're going to need to start the HTTP Server Capability Provider. Actors are signed WebAssembly modules that do not have access to create or receive HTTP requests, so the capability provider will do that for us and deliver messages to and from the actor. 

```

Comment:
this is confusing to me, the `do not have access` bit makes me think actors don't have access to HTTP providers, but they do once we sign them. Maybe rephrase this a bit so it's a bit more clear that "Actors are signed WebAssembly modules, and they can have embedded claims declaring their ability to communicate with capability providers like the HTTP server" (or whatever)